source/about_me.html.haml

```diff
@@ -1,20 +1,20 @@
 .title
   %h1 About me
   %br
-  %p After studying Animal Science, Veterinary medicine and achieving a master's degree in Molecular Biology, I've decided to switch to Web Development at age of 27 to follow the recent dream of becoming a programmer.
+  %p After studying Animal Science, Veterinary medicine and achieving a master's degree in Molecular Biology, I've decided to switch to Web Development at age 27 to follow my recent dream of becoming a programmer.
   %br
-  %p My field of expertise is originally biology and I am very passionate about cultures of bacteria, yeast and virus, DNA sequencing, all the new and old molecular biology techniques and I still follow the latest updates within the scientific community.
+  %p My field of expertise is originally biology and I am very passionate about cultures of bacteria, yeast &amp; viruses, DNA sequencing; all the new &amp; old molecular biology techniques and I still follow the latest updates within the scientific community.
   %br
-  %p I've started my journey into programming last year with my first Java book and from that moment I've been anchored to this world driven by a really strong passion for coding and creating programs. After trying to make a living out of my passion for Life Science going through many difficulties, the bootcamp for me has been the opportunity I needed to give myself a second chance at working with something I love.
+  %p I started my journey into programming last year with my first Java book and from that moment I've been anchored to this world driven by a really strong passion for coding and creating programs. After having many difficulties trying to make a living out of my passion for Life Sciences, the bootcamp for me has been the opportunity I needed to give myself a second chance at working with something I love.
```

Comment:
it's an interesting point you bring up here about difficulties in the life sciences.  By not saying what kind of difficulties you maybe leave too much to the imagination ...